v2/tools/generator/internal/codegen/code_generator.go

```diff
@@ -301,7 +302,8 @@ func (generator *CodeGenerator) Generate(
 			stageDescription,
 			"elapsed", duration,
 			"added", len(defsAdded),
-			"removed", len(defsRemoved))
+			"removed", len(defsRemoved),
+			"count", len(newState.Definitions()))
```

Comment:
minor: Would `total` be any clearer here? TBH if I couldn't see the variable names it might not be super clear to me what these numbers were - added/removed/count of what? Maybe we should consider putting a suffix like `addedDefs` or similar?

Reply:
Changed to `totalDefs`; I don't think `added` and `removed` need a suffix too, the implied context should be enough.